src/main/app/case/from-api-format.ts

```diff
@@ -69,4 +76,7 @@ const fromApiDate = date => {
   return { year: `${+y}`, month: `${+m}`, day: `${+d}` };
 };
 
-export const fromApiFormat = (data: CaseData): Case => formatCase(fields, data);
+export const fromApiFormat = (data: CaseData): Case => ({
+  ...formatCase(fields, data),
+  ...formatCase(invert(readOnlyFormFieldsToCaseMapping), data),
```

Comment:
1. maybe you only need to call the method once with a new constructed object e.g. 
formatCase({ ...fields, ...invert(readOnlyFormFieldsToCaseMapping)}, data),
2. doesnt seem like theres a need to invert it but i guess for future proof?

Reply:
:+1: Good spot, I've refactored to this and removed the unneeded `invert`